todo_app/templates/index.html

```diff
@@ -7,11 +7,19 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+  <from action="add Todo" method="post">
+    <label for="name">Enter New Todo: </label>
+    <input type="text" name="title" required />
+    <input type="submit" value="Add New Todo" />
+    </from>
+
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+          <1i>{{ item.id }} - {{ item.title }} - {{ item.status }}</1i>
```

Comment:
Another subtle typing mismatch here - we want `<li>` (standing for "list item") rather than `<1i>` (i.e. a lowercase L rather than a 1) :
```suggestion
          <li>{{ item.id }} - {{ item.title }} - {{ item.status }}</li>
```